test/spec/algoliasearch.helper/distinct.js

```diff
@@ -1,69 +1,66 @@
 'use strict';
 
-var test = require('tape');
 var forEach = require('lodash/forEach');
 
 var algoliasearchHelper = require('../../../index.js');
 var requestBuilder = require('../../../src/requestBuilder');
 
 var fakeClient = {};
 
-test('Distinct not set', function(t) {
+test('Distinct not set', function() {
   var helper = algoliasearchHelper(fakeClient, null, {
     facets: ['facetConj'],
     disjunctiveFacets: ['facet']
   });
   var state0 = helper.state;
 
   var disjunctiveFacetSearchParam = requestBuilder._getDisjunctiveFacetSearchParams(helper.state);
-  t.equal(disjunctiveFacetSearchParam.distinct, undefined, '[disjunctive] distinct should be undefined');
+  expect(disjunctiveFacetSearchParam.distinct).toBe(undefined);
   var facetSearchParam = requestBuilder._getHitsSearchParams(helper.state);
-  t.equal(facetSearchParam.distinct, undefined, '[hits] distinct should be undefined');
+  expect(facetSearchParam.distinct).toBe(undefined);
 
   helper.setState(state0);
   helper.setQuery('not empty');
   disjunctiveFacetSearchParam = requestBuilder._getDisjunctiveFacetSearchParams(helper.state);
-  t.equal(disjunctiveFacetSearchParam.distinct, undefined, '[disjunctive][query not empty] distinct should be undefined');
+  expect(disjunctiveFacetSearchParam.distinct).toBe(undefined);
   facetSearchParam = requestBuilder._getHitsSearchParams(helper.state);
-  t.equal(facetSearchParam.distinct, undefined, '[hits][query not empty] distinct should be undefined');
+  expect(facetSearchParam.distinct).toBe(undefined);
   forEach(requestBuilder._getQueries('', helper.state), function(q) {
-    t.notOk(q.hasOwnProperty('distinct'), '[hits][query not empty] no distinct should be in the queries by default');
+    expect(q.hasOwnProperty('distinct')).toBeFalsy();
```

Comment:
We can be more strict.

Reply:
See https://github.com/algolia/algoliasearch-helper-js/pull/660#discussion_r278628453